<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="15" passed="6" failed="9" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-09-22T12:29:11Z" name="Suite2" finished-at="2021-09-22T12:30:32Z" duration-ms="81136">
    <groups>
    </groups>
    <test started-at="2021-09-22T12:29:11Z" name="Test" finished-at="2021-09-22T12:30:32Z" duration-ms="81136">
      <class name="com.orangehrm.testcases.login.LT01_LoginPageTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:29:11Z" name="setup" finished-at="2021-09-22T12:29:17Z" duration-ms="5483" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyLogo()[pri:1, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:29:17Z" name="verifyLogo" finished-at="2021-09-22T12:29:17Z" duration-ms="194" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLogo -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:29:17Z" name="tearDown" finished-at="2021-09-22T12:29:17Z" duration-ms="450" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:29:17Z" name="setup" finished-at="2021-09-22T12:29:23Z" duration-ms="5374" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="loginURLTest()[pri:2, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:29:23Z" name="loginURLTest" finished-at="2021-09-22T12:29:26Z" duration-ms="3691" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginURLTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:29:26Z" name="tearDown" finished-at="2021-09-22T12:29:27Z" duration-ms="324" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:29:27Z" name="setup" finished-at="2021-09-22T12:29:32Z" duration-ms="4824" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="loginTest(java.lang.String, java.lang.String)[pri:3, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:29:32Z" name="loginTest" data-provider="Credentials1" finished-at="2021-09-22T12:29:32Z" duration-ms="28" status="FAIL">
          <params>
            <param index="0">
              <value is-null="true"/>
            </param>
            <param index="1">
              <value is-null="true"/>
            </param>
          </params>
          <exception class="java.lang.IllegalArgumentException">
            <message>
              <![CDATA[Keys to send should be a not null CharSequence]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.IllegalArgumentException: Keys to send should be a not null CharSequence
at org.openqa.selenium.remote.RemoteWebElement.sendKeys(RemoteWebElement.java:97)
at com.orangehrm.testcases.login.LT01_LoginPageTest.loginTest(LT01_LoginPageTest.java:39)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.IllegalArgumentException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:29:32Z" name="tearDown" finished-at="2021-09-22T12:29:32Z" duration-ms="113" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:29:32Z" name="setup" finished-at="2021-09-22T12:29:37Z" duration-ms="4894" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="loginTest(java.lang.String, java.lang.String)[pri:3, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:29:37Z" name="loginTest" data-provider="Credentials1" finished-at="2021-09-22T12:29:40Z" duration-ms="3527" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[Admin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[N!dhi@12#]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:29:40Z" name="tearDown" finished-at="2021-09-22T12:29:41Z" duration-ms="622" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:29:41Z" name="setup" finished-at="2021-09-22T12:29:47Z" duration-ms="6416" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="loginTest(java.lang.String, java.lang.String)[pri:3, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:29:47Z" name="loginTest" data-provider="Credentials1" finished-at="2021-09-22T12:29:48Z" duration-ms="1302" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Admin1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[N!dhi@12#]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Both the url's did not match, so test failed expected [http://localhost/orangehrm/symfony/web/index.php/dashboard] but found [http://localhost/orangehrm/symfony/web/index.php/auth/validateCredentials]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Both the url's did not match, so test failed expected [http://localhost/orangehrm/symfony/web/index.php/dashboard] but found [http://localhost/orangehrm/symfony/web/index.php/auth/validateCredentials]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at com.orangehrm.testcases.login.LT01_LoginPageTest.loginTest(LT01_LoginPageTest.java:44)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:29:48Z" name="tearDown" finished-at="2021-09-22T12:29:49Z" duration-ms="126" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:29:49Z" name="setup" finished-at="2021-09-22T12:29:54Z" duration-ms="5232" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="loginTest(java.lang.String, java.lang.String)[pri:3, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:29:54Z" name="loginTest" data-provider="Credentials1" finished-at="2021-09-22T12:29:56Z" duration-ms="1798" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Admin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[N!dhi@12]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Both the url's did not match, so test failed expected [http://localhost/orangehrm/symfony/web/index.php/dashboard] but found [http://localhost/orangehrm/symfony/web/index.php/auth/validateCredentials]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Both the url's did not match, so test failed expected [http://localhost/orangehrm/symfony/web/index.php/dashboard] but found [http://localhost/orangehrm/symfony/web/index.php/auth/validateCredentials]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at com.orangehrm.testcases.login.LT01_LoginPageTest.loginTest(LT01_LoginPageTest.java:44)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:29:56Z" name="tearDown" finished-at="2021-09-22T12:29:56Z" duration-ms="124" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:29:56Z" name="setup" finished-at="2021-09-22T12:30:01Z" duration-ms="5077" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="loginTest(java.lang.String, java.lang.String)[pri:3, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:30:01Z" name="loginTest" data-provider="Credentials1" finished-at="2021-09-22T12:30:02Z" duration-ms="1200" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Admin12]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[N!dhi@123]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Both the url's did not match, so test failed expected [http://localhost/orangehrm/symfony/web/index.php/dashboard] but found [http://localhost/orangehrm/symfony/web/index.php/auth/validateCredentials]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Both the url's did not match, so test failed expected [http://localhost/orangehrm/symfony/web/index.php/dashboard] but found [http://localhost/orangehrm/symfony/web/index.php/auth/validateCredentials]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at com.orangehrm.testcases.login.LT01_LoginPageTest.loginTest(LT01_LoginPageTest.java:44)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:30:02Z" name="tearDown" finished-at="2021-09-22T12:30:02Z" duration-ms="120" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:30:02Z" name="setup" finished-at="2021-09-22T12:30:08Z" duration-ms="5855" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="loginTest(java.lang.String, java.lang.String)[pri:3, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:30:08Z" name="loginTest" data-provider="Credentials1" finished-at="2021-09-22T12:30:08Z" duration-ms="387" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Admin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Both the url's did not match, so test failed expected [http://localhost/orangehrm/symfony/web/index.php/dashboard] but found [http://localhost/orangehrm/symfony/web/index.php/auth/login]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Both the url's did not match, so test failed expected [http://localhost/orangehrm/symfony/web/index.php/dashboard] but found [http://localhost/orangehrm/symfony/web/index.php/auth/login]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at com.orangehrm.testcases.login.LT01_LoginPageTest.loginTest(LT01_LoginPageTest.java:44)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:30:08Z" name="tearDown" finished-at="2021-09-22T12:30:08Z" duration-ms="143" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:30:08Z" name="setup" finished-at="2021-09-22T12:30:17Z" duration-ms="9004" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="loginTest(java.lang.String, java.lang.String)[pri:3, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:30:17Z" name="loginTest" data-provider="Credentials1" finished-at="2021-09-22T12:30:18Z" duration-ms="408" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[N!dhi@12#]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Both the url's did not match, so test failed expected [http://localhost/orangehrm/symfony/web/index.php/dashboard] but found [http://localhost/orangehrm/symfony/web/index.php/auth/login]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Both the url's did not match, so test failed expected [http://localhost/orangehrm/symfony/web/index.php/dashboard] but found [http://localhost/orangehrm/symfony/web/index.php/auth/login]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at com.orangehrm.testcases.login.LT01_LoginPageTest.loginTest(LT01_LoginPageTest.java:44)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.orangehrm.testcases.login.LT01_LoginPageTest@e3cee7b]" started-at="2021-09-22T12:30:18Z" name="tearDown" finished-at="2021-09-22T12:30:18Z" duration-ms="420" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.orangehrm.testcases.login.LT01_LoginPageTest -->
      <class name="com.orangehrm.testcases.login.LT10_HomePageTest">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.orangehrm.testcases.login.LT10_HomePageTest@71e9a896]" started-at="2021-09-22T12:30:18Z" name="setup" finished-at="2021-09-22T12:30:24Z" duration-ms="5209" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="adminTabTest()[pri:4, instance:com.orangehrm.testcases.login.LT10_HomePageTest@71e9a896]" started-at="2021-09-22T12:30:24Z" name="adminTabTest" finished-at="2021-09-22T12:30:32Z" duration-ms="8476" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- adminTabTest -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.orangehrm.testcases.login.LT10_HomePageTest@71e9a896]" started-at="2021-09-22T12:30:32Z" name="tearDown" finished-at="2021-09-22T12:30:32Z" duration-ms="232" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.orangehrm.testcases.login.LT10_HomePageTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite2 -->
  <suite started-at="2021-09-22T12:30:32Z" name="Suite3" finished-at="2021-09-22T12:31:45Z" duration-ms="72494">
    <groups>
    </groups>
    <test started-at="2021-09-22T12:30:32Z" name="Test" finished-at="2021-09-22T12:31:45Z" duration-ms="72494">
      <class name="com.orangehrm.testcases.mailConfiguration.MT05_Reset_Config_Settings">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.orangehrm.testcases.mailConfiguration.MT05_Reset_Config_Settings@5be82d43]" started-at="2021-09-22T12:30:49Z" name="setup" finished-at="2021-09-22T12:30:55Z" duration-ms="6426" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="ResetConfigSettings()[pri:0, instance:com.orangehrm.testcases.mailConfiguration.MT05_Reset_Config_Settings@5be82d43]" started-at="2021-09-22T12:30:55Z" name="ResetConfigSettings" finished-at="2021-09-22T12:31:06Z" duration-ms="10931" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ResetConfigSettings -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.orangehrm.testcases.mailConfiguration.MT05_Reset_Config_Settings@5be82d43]" started-at="2021-09-22T12:31:06Z" name="tearDown" finished-at="2021-09-22T12:31:07Z" duration-ms="233" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.orangehrm.testcases.mailConfiguration.MT05_Reset_Config_Settings -->
      <class name="com.orangehrm.testcases.mailConfiguration.MT01_MatchURL">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.orangehrm.testcases.mailConfiguration.MT01_MatchURL@82c57b3]" started-at="2021-09-22T12:30:32Z" name="setup" finished-at="2021-09-22T12:30:37Z" duration-ms="5090" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="MatchConfigModuleURL()[pri:0, instance:com.orangehrm.testcases.mailConfiguration.MT01_MatchURL@82c57b3]" started-at="2021-09-22T12:30:37Z" name="MatchConfigModuleURL" finished-at="2021-09-22T12:30:49Z" duration-ms="11435" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- MatchConfigModuleURL -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.orangehrm.testcases.mailConfiguration.MT01_MatchURL@82c57b3]" started-at="2021-09-22T12:30:49Z" name="tearDown" finished-at="2021-09-22T12:30:49Z" duration-ms="133" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.orangehrm.testcases.mailConfiguration.MT01_MatchURL -->
      <class name="com.orangehrm.testcases.mailConfiguration.MT02_ConfigDetailsValidation">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.orangehrm.testcases.mailConfiguration.MT02_ConfigDetailsValidation@600b0b7]" started-at="2021-09-22T12:31:07Z" name="setup" finished-at="2021-09-22T12:31:12Z" duration-ms="5061" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Add_Configurations(java.lang.String)[pri:0, instance:com.orangehrm.testcases.mailConfiguration.MT02_ConfigDetailsValidation@600b0b7]" started-at="2021-09-22T12:31:12Z" name="Add_Configurations" data-provider="testdata" finished-at="2021-09-22T12:31:20Z" duration-ms="7642" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[patninidhi98@gmail.com]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Required is required in given format  expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Required is required in given format  expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at com.orangehrm.testcases.mailConfiguration.MT02_ConfigDetailsValidation.Add_Configurations(MT02_ConfigDetailsValidation.java:44)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Add_Configurations -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.orangehrm.testcases.mailConfiguration.MT02_ConfigDetailsValidation@600b0b7]" started-at="2021-09-22T12:31:20Z" name="tearDown" finished-at="2021-09-22T12:31:20Z" duration-ms="136" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:com.orangehrm.testcases.mailConfiguration.MT02_ConfigDetailsValidation@600b0b7]" started-at="2021-09-22T12:31:20Z" name="setup" finished-at="2021-09-22T12:31:25Z" duration-ms="4871" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Add_Configurations(java.lang.String)[pri:0, instance:com.orangehrm.testcases.mailConfiguration.MT02_ConfigDetailsValidation@600b0b7]" started-at="2021-09-22T12:31:25Z" name="Add_Configurations" data-provider="testdata" finished-at="2021-09-22T12:31:32Z" duration-ms="7427" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[patninidhi98@gmail]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Required is required in given format  expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Required is required in given format  expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at com.orangehrm.testcases.mailConfiguration.MT02_ConfigDetailsValidation.Add_Configurations(MT02_ConfigDetailsValidation.java:44)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Add_Configurations -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.orangehrm.testcases.mailConfiguration.MT02_ConfigDetailsValidation@600b0b7]" started-at="2021-09-22T12:31:32Z" name="tearDown" finished-at="2021-09-22T12:31:32Z" duration-ms="146" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method is-config="true" signature="setup()[pri:0, instance:com.orangehrm.testcases.mailConfiguration.MT02_ConfigDetailsValidation@600b0b7]" started-at="2021-09-22T12:31:32Z" name="setup" finished-at="2021-09-22T12:31:37Z" duration-ms="5086" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="Add_Configurations(java.lang.String)[pri:0, instance:com.orangehrm.testcases.mailConfiguration.MT02_ConfigDetailsValidation@600b0b7]" started-at="2021-09-22T12:31:37Z" name="Add_Configurations" data-provider="testdata" finished-at="2021-09-22T12:31:45Z" duration-ms="7467" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Required is required in given format  expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Required is required in given format  expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at com.orangehrm.testcases.mailConfiguration.MT02_ConfigDetailsValidation.Add_Configurations(MT02_ConfigDetailsValidation.java:44)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Add_Configurations -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.orangehrm.testcases.mailConfiguration.MT02_ConfigDetailsValidation@600b0b7]" started-at="2021-09-22T12:31:45Z" name="tearDown" finished-at="2021-09-22T12:31:45Z" duration-ms="113" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.orangehrm.testcases.mailConfiguration.MT02_ConfigDetailsValidation -->
    </test> <!-- Test -->
  </suite> <!-- Suite3 -->
</testng-results>
